Source/WebKit/UIProcess/API/Swift/WebView.swift

```diff
@@ -115,6 +120,18 @@ fileprivate struct WebViewRepresentable {
         webView.configuration.preferences.isTextInteractionEnabled = environment.webViewAllowsTextInteraction
         webView.configuration.preferences.tabFocusesLinks = environment.webViewAllowsTabFocusingLinks
         webView.configuration.preferences.isElementFullscreenEnabled = environment.webViewAllowsElementFullscreen
+
+        if let cameraCaptureState = environment.webViewCameraCaptureState, cameraCaptureState.wrappedValue != webView.cameraCaptureState {
+            webView.setCameraCaptureState(cameraCaptureState.wrappedValue)
+        }
+
+        if let microphoneCaptureState = environment.webViewMicrophoneCaptureState, microphoneCaptureState.wrappedValue != webView.microphoneCaptureState {
+            webView.setMicrophoneCaptureState(microphoneCaptureState.wrappedValue)
+        }
+    }
+
+    func makeCoordinator() -> Coordinator {
+        Coordinator(cameraCaptureState: cameraCaptureState, microphoneCaptureState: microphoneCaptureState)
```

Comment:
These bindings can refresh, so you might end up with stale values if you only take them in during object creation.